browser-test/src/support/admin_questions.ts

```diff
@@ -226,35 +226,15 @@ export class AdminQuestions {
     )
   }
 
-  private async gotoQuestionEditOrNewVersionPage({
-    questionName,
-    buttonText,
-  }: {
-    questionName: string
-    buttonText: string
-  }) {
+  async gotoQuestionEditPage(questionName: string) {
     await this.gotoAdminQuestionsPage()
     await this.page.click(
-      this.selectWithinQuestionTableRow(questionName, `:text("${buttonText}")`),
+      this.selectWithinQuestionTableRow(questionName, `:text("Edit")`),
```

Comment:
Not blocking: Can we add the -> character here without breaking tests?

Reply:
Updated!

Reply:
Ok, that worked on the local sample test I ran, but caused some other tests to break so I'm rescinding it.